pyleecan/Methods/Slot/HoleM50/comp_magnetization_dict.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+from numpy import pi
+from pyleecan.Classes.Segment import Segment
+
+
+def comp_magnetization_dict(self, is_north=True, return_type=2):
+    """Compute the dictionary of the magnetization direction of the magnets (key=magnet_X, value=angle[rad])
+    Mangetization angle with Hole centered on Ox axis
+
+    Parameters
+    ----------
+    self : HoleM50
+        a HoleM50 object
+    is_north: True
+        True: comp north magnetization, else add pi [rad]
+    return_type : int
+        0: Normal as tuple of complex
+        1: Z2 - Z1 (with Z1 and Z2 the points from return_type 0)
+        2: the angle of the vector of return_type 1 according to Ox [rad]
```

Comment:
See 'comp_normal'.
If you keep the current implementation with retrun_type, the return_type isn't checked if enforced dict is not None (despite in Hole comp_mag. method).

Reply:
Same, I think that the cleanest solution would be to remove return_type.